support/nrsc5.py

```diff
@@ -29,6 +29,12 @@ class EventType(enum.Enum):
     PACKET = 13
 
 
+class SampleRates(float, enum.Enum):
+    CU8 = 1488375
+    CS16 = 744187.5
+    AUDIO = 44100
```

Comment:
Grouping these into an `Enum` seems clumsy, since it takes extra work to get at the underlying values. Maybe it would be easier to make them contants:

```suggestion
SAMPLE_RATE_CU8 = 1488375
SAMPLE_RATE_CS16 = 744187.5
SAMPLE_RATE_AUDIO = 44100
```